pkg/agent/cniserver/pod_configuration.go

```diff
@@ -56,7 +56,6 @@ const (
 
 const (
 	defaultOVSInterfaceType int = iota //nolint suppress deadcode check for windows
```

Comment:
Remove `defaultOVSInterfaceType` as it is not used any more?

Reply:
Removed.